website/src/docs/hotchocolate/v12/api-reference/apollo-federation.md

```diff
@@ -2,6 +2,705 @@
 title: Apollo Federation Subgraph Support
 ---
 
-> Note: Apollo Federation Support is coming with Hot Chocolate 12.6
+> If you want to read more about Apollo Federation in general, you can head over to the [Apollo Federation documentation](https://www.apollographql.com/docs/federation/), which provides a robust overview and set of examples for this GraphQL architectural pattern. Many of the core principles and concepts are referenced within this document.
 
-## Example subgraphs
+Hot Chocolate includes an implementation of the Apollo Federation v1 specification for creating Apollo Federated subgraphs. Through Apollo Federation, you can combine multiple GraphQL APIs into a single API for your consumers.
+
+The documentation describes the syntax for creating an Apollo Federated subgraph using Hot Chocolate and relates the implementation specifics to its counterpart in the Apollo Federation docs. This document _will not_ provide a thorough explanation of the Apollo Federation core concepts nor will it describe how you go about creating a supergraph to stitch together various subgraphs, as the Apollo Federation team already provides thorough documentation of those principles.
+
+You can find example projects of the Apollo Federation library in [Hot Chocolate examples](https://github.com/ChilliCream/graphql-platform/tree/main/src/HotChocolate/ApolloFederation/examples).
+
+# Get Started
+
+To use the Apollo Federation tools, you need to first install v12.6 or later of the `HotChocolate.ApolloFederation` package.
+
+<PackageInstallation packageName="HotChocolate.ApolloFederation"/>
+
+After installing the necessary package, you'll need to register the Apollo Federation services with the GraphQL server.
+
+```csharp
+IServiceCollection services;
+
+services.AddGraphQLServer()
+    .AddApolloFederation();
+```
+
+# Defining an entity
+
+Now that the API is ready to support Apollo Federation, we'll need to define an **entity**&mdash;an object type that can resolve its fields across multiple subgraphs. We'll work with a `Product` entity to provide an example of how to do this.
+
+<ExampleTabs>
+<Annotation>
+
+  ```csharp
+  public class Product
+  {
+      [GraphQLType(typeof(NonNullType<IdType>))]
```

Comment:
Couldn't this (and others below) be the `[ID]` attribute? Simplifies the declaration

Reply:
Oops! I had no idea that attribute existed 🤦 Yeah that would help make the code a little more succinct. Thanks for improving my Hot Chocolate knowledge today! I'll clean that up so that things are a bit neater 👍

Reply:
Resolved in 623bd3cc4c8c37ddda42fb40f339942a3f77c9d4